src/news.njk

```diff
@@ -18,7 +18,7 @@ pagination:
 			{% endfor %}
 		</div>
 		{% if pagination.pages.length > 1 %}
-		  {% include 'components/pagination.njk' %}
+			{% include 'components/pagination.njk' %}
```

Comment:
What was changed in this line?

Reply:
The change here is to replace spaces with a tab.